cmd/soroban-cli/src/commands/contract/invoke.rs

```diff
@@ -315,6 +315,8 @@ impl NetworkRunnable for Cmd {
     type Error = Error;
     type Result = TxnResult<String>;
 
+    //TODO: remove
+    #[allow(clippy::map_clone)]
```

Comment:
what's this?

Reply:
This was my bad. There was an update to clippy (which I think CI isn't using yet) and locally it was yelling at me. I'll remove it as it should be part of another PR when it's actually a blocker.